tool/logger/implementation/zap/logger.go

```diff
@@ -408,6 +411,8 @@ func (l *CompactLogger) Log(level types.Level, values ...any) {
 		buf := l.acquireBuf()
 		defer l.releaseBuf(buf)
 		valuesParser.ForEachField(func(f *field.Field) bool { return true }) // just causing to parse the fields to make valuesParser.WriteUnparsed(buf) work correctly
+		buf.WriteString(l.messagePrefix)
+		valuesParser.WriteUnparsed(buf)
```

Comment:
This line also fixes another bug (empty message in zap with hooks using method Log)